src/open_producten/producten/tests/test_product.py

```diff
@@ -53,7 +53,9 @@ def test_kvk_validation_validates_on_valid_value(self):
     def test_bsn_or_kvk_required(self):
         product = ProductFactory.build(product_type=self.product_type)
 
-        with self.assertRaises(ValidationError):
+        with self.assertRaisesMessage(
+            ValidationError, "Een product moet een bsn, kvk nummer of beiden hebben."
```

Comment:
```suggestion
            ValidationError, _("Een product moet een bsn, kvk nummer of beiden hebben.")
```
Since the error is translated, so should the expected result